assignment/assignment-tool/tool/src/webapp/js/studentViewSubmission.js

```diff
@@ -31,3 +31,25 @@ ASN_SVS.undoCancel = function()
 	submitPanel.style.display = "block";
 	confirmationDialogue.style.display = "none";
 };
+
+$(document).ready(function()
+{
+	var inTool = false;
+
+	window.addEventListener("click", function (event) {
+		if ($.contains(document.getElementById('addSubmissionForm'), event.target)) {
```

Comment:
`inTool = $.contains(document.getElementById('addSubmissionForm'), event.target);` is a more concise way to write this.